exa.py

```diff
@@ -54,6 +54,19 @@ def do(self, interp_state):
             interp_state.next_statement += 1
 
 
+class JUMP(Statement):
+
+    _expected_args = 2
+
+    def __init__(self, line_num, tokens):
+        super().__init__(line_num, tokens)
+        self._label = tokens[1]
+
+    def do(self, interp_state):
+        label_address = interp_state.labels[self._label]
```

Comment:
In most of the rest of the places where errors can occur I tried to produce an error message that showed the line number of the input that has the bad value. We should catch the KeyError here and produce a ValueError or RuntimeError.